spark/src/main/scala/io/delta/tables/DeltaMergeBuilder.scala

```diff
@@ -259,6 +263,22 @@ class DeltaMergeBuilder private(
     DeltaMergeNotMatchedBySourceActionBuilder(this, Some(condition))
   }
 
+  /**
```

Comment:
This API is optional right? If so, should we add it to the description?

Reply:
I checked other places, and WHEN clauses such as `whenNotMatched` and `whenNotMatchedBySource` are always optional, but their docstring does not mention that, so I think not mentioning it is more common.